test/StakeManager.js

```diff
@@ -1204,5 +1205,43 @@ describe('StakeManager', function () {
       const tx = stakeManager.connect(signers[3]).withdraw(epoch, stakerIdacc3);
       await assertRevert(tx, 'Nonpositive Stake');
     });
+
+    it('staker should not be able to withdraw more than his sToken balance', async function () {
```

Comment:
same

Reply:
> > As per discussion in testing call we decided to remove both require statements from both functions as they are already been checked before reaching them.
> 
> Please provide a detailed justification as to why they are removed with code, links etc.

Provided the detailed justification here it is also provided in issues and the link to the same was already given.